resin/models/build.py

```diff
@@ -43,6 +43,25 @@ def get(self, id):
         else:
             raise exceptions.BuildNotFound(id)
 
+    def get_by_commit(self, commit_hash):
+        """
+        Get a specific build by commit hash.
```

Comment:
I think the wording isn't quite right: there can easily be multiple builds for a commit hash (just only one successful build), so this should be something like `Get all builds for a commit hash`

Reply:
@pimterry updated!